sources/packages/web/src/services/StudentService.ts

```diff
@@ -119,8 +119,10 @@ export class StudentService {
    * @param studentId
    * @returns StudentDetail
    */
-  async getStudentDetail(studentId: number): Promise<StudentDetail> {
-    return ApiClient.Students.getStudentDetail(studentId);
+  async getAESTStudentProfile(
```

Comment:
The get method can be named irrespective of client type(getStudentProfile) with studentId as optional parameter.

Reply:
Yes, it can, but should we do it?
The reasons that I kept two are:
1. We need to adapt the URL for one or another if the student id is present or not;
2. The AEST one can possibly return more data in the future, like SIN number, which would make the method return two DTOs.
Considering the above points, do you believe it would still be worth keeping only one?
I am ok to change it if it is a Team understanding. @ann-aot @guru-aot @henryoforeh-aot - FYI

Reply:
I believe, I remember a similar comment on the renaming of the method name in the API service side, I changed it. But here this is in the web side and I think it would be better to have it irrespective of the client, until we have a requirement for it. Just my thoughts 😄

Reply:
Agree with @guru-aot . If the difference between both both API calls are growing wider than just studentId/InstitutionId as optional param, then we can start using different client service methods.

Reply:
Method is centralized.